molecule/cookiecutter/scenario/driver/ec2/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -29,41 +30,63 @@
     keypair_name: molecule_key
     keypair_path: "{{ lookup('env', 'MOLECULE_EPHEMERAL_DIRECTORY') }}/ssh_key"
   tasks:
+    - name: Check subnet
+      ec2_vpc_subnet_facts:
+        subnet_ids: "{{ item.vpc_subnet_id }}"
```

Comment:
This is a total disaster due to https://github.com/ansible/ansible/issues/60949

Use approach below which is not blocked by the exponential backoff bug:
```
 filters:
            subnet-id: "{{ item }}"
```

Reply:
thank you! fixed